tests/components/FieldValueDirectAnswer.test.tsx

```diff
@@ -0,0 +1,168 @@
+import { cleanup, render, screen } from '@testing-library/react';
+import { FieldValueDirectAnswer } from '../../src/components/FieldValueDirectAnswer';
+import { Address, BuiltInFieldType, DirectAnswerType, FieldValueDirectAnswer as FieldValueDirectAnswerType, Source } from '@yext/search-headless-react';
+import { userEvent } from '@storybook/testing-library';
+
+const fieldValueDAResult: FieldValueDirectAnswerType = {
```

Comment:
can this be moved to fixtures? I think we already have a field value example there?